### ISSUE TITLE
Could somebody add function Percent() to divide by hundred or decimal.Hundred?

### ISSUE THREAD

**Issue by aigoya** [shopspring/decimal#259]
I think one and hundred both of them are used a lot in the application (my application).
Example I want to find Price after Discount I have to write:
```go
Price.Sub(Price.Mul(Discount.Div(decimal.New(100, 0))) // hard to read
```
or
```go
Price.Mul(decimal.New(1, 0).Sub(Discount.Div(decimal.New(100, 0)))) // harder to read.
```

Case 1:
We already have decimal.Zero. Could we add decimal.One and decimal.Hundred.
```go
Price.Mul(decimal.One.Sub(Discount.Div(decimal.Hundred))) // easier
```
Case 2:
Add function name Percent or something equivalent 
```go
Price.Sub(Price.Mul(Discount.Percent())) // much easier
```
Thanks!!!

**Comment by mwoss** [comment]
Not sure if this library needs the` Percent` method, in my opinion, such API design would be confusing, and saving a few keystrokes is not the best tradeoff here.
On the other hand, a module, containing a few or even more helper constants would be a great addition, I will add this idea to our backlog :).

**Comment by rickb777** [comment]
Surely `Shift(-2)` and `Shift(2)` are better expressions for converting to/from percent.

**Comment by mwoss** [comment]
No, they are not. I just think the `Percentage` method should not be a part of the decimal struct as it may be confusing in many scenarios. In the example given by OP, it makes a lot of sense, but what if we have a decimal that stores transaction amount? We would still have access to the `Percentage` method there, and it would be quite non-intuitive. 
Of course, there are other decimal methods that are pointless in the scope of such variable (like Sin, Tan, etc), but calculating them is not trivial, that's why they are part of the decimal structure.
Personally, I believe that percentage values should be just stored as raw decimal values or calculated explicitly to decrease the potential area of errors.
I could be wrong with my reasoning tho, so we can discuss this topic. Just please stop being sarcastic it does not help with the overall conversation.
